process-automation/latest/develop/index.md

```diff
@@ -93,6 +88,530 @@ Once the extended Digital Workspace has been fully customized and tested it can
+
+
+```typescript
+@NgModule({
+    declarations: [TaskNameComponent, TaskPriorityComponent, TaskDueDateComponent ]
+})
+
+export class TasksListCloudModule {
+    constructor(extensions: ExtensionService) {
+        extensions.setComponents({
+            'app.taskList.columns.name': TaskNameComponent,
+            'app.taskList.columns.priority': TaskPriorityComponent,
+            'app.taskList.columns.dueDate': TaskDueDateComponent
+        });
+    }
+}
+```
+
+Once you have registered your components, you need to register your new template component. To do this you need to add your new column to the `your-app.extensions.json` file:
+
+```json
+{
+  "features": {
+    "taskList": {
+        "presets": {
+            "default": [
+                {
+                  "id": "app.task.name",
+                  "key": "name",
+                  "title": "Name",
+                  "type": "text",
+                  "template": "app.taskList.columns.name",
+                  "sortable": true
+                },
+                {
+                  "id": "app.task.dueDate",
+                  "key": "status",
+                  "title": "Due Date",
+                  "type": "text",
+                  "template": "app.taskList.columns.dueDate",
+                  "sortable": true
+                },
+                {
+                  "id": "app.task.priority",
+                  "key": "priority",
+                  "title": "Priority",
+                  "type": "text",
+                  "template": "app.taskList.columns.priority",
+                  "sortable": true
+                }
+            ]
+        }
+    }
+  }
+}
+```
+
+Restart the application and you will see the custom columns in the task list based on the new custom template.
+
+![Task list with custom template]({% link process-automation/images/task-list-custom-template.png %})
+
+### Extend process list with custom columns
+
+The Digital Workspace provides a default group of columns for your process list. The columns can be changed and configured in the `process-services-cloud.extension.json` file of the [Process Services Cloud extension library](https://github.com/Alfresco/alfresco-apps/tree/develop/libs/content-ee/process-services-cloud-extension){:target="_blank"}.
+
+Process list with default columns preset.
+
+```json
+{
+  "features": {
+    "processList": {
+        "presets": {
+            "default": [
+                columns schema
+            ]
+        }
+    }
+  }
+}
+```
+
+![Process list with default columns]({% link process-automation/images/process-list-default-columns.png %})
+
+### Add a column in the process list using a process instance property
+
+To display the process list with new columns, edit the `process-services-cloud.extension.json` file, and insert an entry into the `features.processList.presets.default` section.
+
+Process list with new columns.
```

Comment:
Similar to an earlier comment, I don't think this statement is needed - it's already mentioned in the previous sentence.

Reply:
changed